src/Doctrine/ODM/MongoDB/Listener/VectorSearchIndexListener.php

```diff
@@ -25,7 +25,7 @@ public function loadClassMetadata(LoadClassMetadataEventArgs $args): void
         // Remove the original search index definition - ODM only supports Atlas Text Search
         $searchIndexes = array_map(
             static function (array $index) {
-                if ($index['name'] !== 'faces' || ($index['type'] ?? null) === 'vectorSearch') {
+                if (($index['type'] ?? null) === 'vectorSearch') {
```

Comment:
Thanks - this was a leftover from the first iteration where I completely overwrote the existing definition.